projects/Mallard/src/helpers/weather-forecast.ts

```diff
@@ -0,0 +1,76 @@
+import { Forecast, AccuWeatherLocation, WeatherForecast } from '../common'
+import { RESULTS } from 'react-native-permissions'
+import Geolocation, {
+    GeolocationResponse,
+} from '@react-native-community/geolocation'
+
+Geolocation.setRNConfiguration({
+    skipPermissionRequests: true,
```

Comment:
add comment to explain that we want to control when the request appears, for UX quality reasons